base/interactiveutil.jl

```diff
@@ -64,7 +64,27 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-edit(f)          = edit(functionloc(f)...)
+function edit(f)
+    ms = methods(f)
```

Comment:
@omus If you follow through the definition of functionloc, you will see that this new definition for `edit` is equally general to the old, and it applies to more than just instances of `Function`.